### ISSUE TITLE
Support escaped new-lines in RC files

### ISSUE THREAD

**Issue by bovine3dom** [cmcaine/tridactyl#552]
Despite what minifiers might have you believe, incantations such as `js tri.browserBg.runtime.getPlatformInfo().then(os=>{const editorcmd = os!="linux" ? "st vim" : "auto"; tri.config.set("editorcmd", editorcmd)})` are quite hard to parse.

**Comment by glacambre** [comment]
An alternative to escaped newlines could be to support `<<EOF mylongstringwithnewlines EOF`. It's what Pentadactyl did and it might be easier to support than escaping newlines.

**Comment by bovine3dom** [comment]
Closing in favour of #789.
